### ISSUE TITLE
Certin mechanical blocks crash game

### ISSUE THREAD

**Issue by MsAlPatina** [age-series/ElectricalAge#288]
Some blocks like the Clutch or fixed shaft crash the game
Trying to place anything described above will crash the game

[latest.log](https://github.com/age-series/ElectricalAge/files/12016100/latest.log)
[crash-2023-07-11_15.53.08-server.txt](https://github.com/age-series/ElectricalAge/files/12016104/crash-2023-07-11_15.53.08-server.txt)

**Comment by jrddunbr** [comment]
```
Caused by: java.lang.NullPointerException
	at mods.eln.node.transparent.TransparentNode.getBlockMetadata(TransparentNode.kt:175) ~[TransparentNode.class:?]
	at mods.eln.node.transparent.TransparentNodeItem.placeBlockAt(TransparentNodeItem.kt:44) ~[TransparentNodeItem.class:?]
	at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103) ~[abh.class:?]
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505) ~[ForgeHooks.class:?]
```
```
java.lang.NullPointerException: Ticking memory connection
	at mods.eln.node.transparent.TransparentNode.getBlockMetadata(TransparentNode.kt:175)
	at mods.eln.node.transparent.TransparentNodeItem.placeBlockAt(TransparentNodeItem.kt:44)
	at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103)
```
```
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
...
	UCHIJAAAA	Eln{1.21.3} [Electrical Age - jrddunbr's build] (Eln-1.21.3.jar) 
```

**Comment by jrddunbr** [comment]
This is caused by the change to the ValueWatchdog code. I'm looking at ways of fixing it. The issue manifests due to uninitialized values for the coordinates during initialization of the element.

**Comment by jrddunbr** [comment]
Fixed by [8577ea8](https://github.com/age-series/ElectricalAge/commit/8577ea88bf8f3057db6f9afcecb97e38604c5093)
